CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Semaphore Stellar
+
+Thank you for considering contributing to this project! We appreciate your time and effort in helping improve our work. Below are guidelines to streamline the contribution process, so please read them carefully. And don’t forget to join our [Telegram](https://t.me/+-9623JNgGjEyNzI5) for updates and community support!
+
+Following these guidelines shows respect for the maintainers’ time, and they’ll reciprocate by giving thoughtful attention to your issues, reviewing your changes, and supporting you in finalizing pull requests.
+
+### Check existing issues
+
+Before you start contributing, please check the [Issue Tracker](https://github.com/ZencypherSolutions/semaphore-stellar-contracts/issues) to see if there are any existing issues that match what you are intending to do. If the issue doesn't exist, please create it.
+
+If you are creating a new issue, please provide a descriptive title and detailed description. If possible, include a code sample or an executable test case demonstrating the expected behavior that is not occurring.
```

Comment:
Add a new paragraph being more explicit that they can create issues for new features which would then be analyzed by the maintainers.